readme/news/20230508-release-2-10.md

```diff
@@ -67,4 +68,4 @@ Our mobile app now has an improved search function that performs text searches i
 
 ## Improved filesystem sync performance
 
-Thanks to the hard work of jd1378, the sync no longer freezes during filesystem synchronisation. We know how frustrating that can be, and we're thrilled to have solved this issue. Getting filesystem sync to work on Android is never easy due to the restrictions put in place by Google, especially since they frequently change, but we're committed to delivering the best possible experience for our Android users.
\ No newline at end of file
+Thanks to the hard work of jd1378, the sync no longer freezes during filesystem synchronisation. We know how frustrating that can be, and we're thrilled to have solved this issue. Getting filesystem sync to work on Android is never easy due to the restrictions put in place by Google, especially since they frequently change, but we're committed to delivering the best possible experience for our Android users.
```

Comment:
I don't know why there's this change. I didn't touch this line.

Reply:
And it doesn't make sense to remove it. Because, it already was there.

Reply:
Can you please remove it though?